src/components/molecules/searchBar/SearchBar.js

```diff
@@ -0,0 +1,22 @@
+import React, {useRef} from 'react'
+import Button from '../../atoms/button/Button'
+import InputText from '../../atoms/textInput/InputText'
+import {getRepoList} from '../../../service/getRepoList'
+
+export default function SearchBar(props) {
+
+  const searchName = useRef();
+
+  const getRepoInfo = async(userName) => {
+    const result = await getRepoList(userName);
+    props.setRepoList(result);
+    props.setUserName(userName);
+  }
+  
+  return (
+    <div className="SearchBar">
+      <InputText placeholder={"검색할 Github User 또는 Organization의 이름을 입력해주세요!"} searchName={searchName} />
+      <Button name="Search" onClick={() => getRepoInfo(searchName.current.value)}/>
+    </div>
+  )

```

Comment:
```suggestion
  const getRepoInfo = useCallback(async () => {
    const userName = searchName.current.value;
    const result = await getRepoList(userName);
    props.setRepoList(result);
    props.setUserName(userName);
  }, []);
  
  return (
    <div className="SearchBar">
      <InputText placeholder={"검색할 Github User 또는 Organization의 이름을 입력해주세요!"} searchName={searchName} />
      <Button name="Search" onClick={getRepoInfo}/>
    </div>
  )
```
@YukJiSoo 님 말씀대로 컴포넌트의 props로 동적으로 함수를 생성해주게 되면 리렌더링 시마다 새로운 함수가 할당되 이를 props로 넘겨받은 컴포넌트가 불필요하게 리렌더링 될 수 있습니다. 여기서는 `getRepoInfo` 함수 안에서 `searchName` 을 참조할 수 있으니 이렇게 바꿔도 괜찮을것 같습니다!